contrib/charts/cert-manager/templates/deployment.yaml

```diff
@@ -4,20 +4,22 @@ metadata:
   name: {{ template "cert-manager.fullname" . }}
   namespace: {{ .Release.Namespace | quote }}
   labels:
-    app: {{ template "cert-manager.name" . }}
-    chart: {{ template "cert-manager.chart" . }}
+    app.kubernetes.io/name: {{ template "cert-manager.name" . }}
+    app.kubernetes.io/managed-by: helm
+    app.kubernetes.io/part-of: {{ template "cert-manager.name" . }}
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
+    chart: {{ template "cert-manager.chart" . }}
 spec:
   replicas: {{ .Values.replicaCount }}
   selector:
     matchLabels:
-      app: {{ template "cert-manager.name" . }}
+      app.kubernetes.io/name: {{ template "cert-manager.name" . }}
```

Comment:
We can't change the selector on existing deployments, so changing this will break the upgrade as far as I'm aware.

Can you update this PR so that the changes are only additive for now? We can then work out a strategy for phasing out the old labels.

Reply:
Hi @munnerz !

By "addictive" you mean adding the new labels, whilst keeping the old ones, right?

Thanks for the feedback! I will proceed with the changes as soon as I get your confirmation.

Reply:
Yep 😄